hydroflow/examples/two_pc/protocol.rs

```diff
@@ -0,0 +1,28 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(PartialEq, Eq, Clone, Serialize, Deserialize, Debug, Hash, Copy)]
+pub enum MsgType {
+    Prepare,
+    Commit,
+    Abort,
+    AckP2,
+    End,
+    Ended,
+    Err,
+}
+
+#[derive(PartialEq, Eq, Clone, Serialize, Deserialize, Debug)]
+pub struct CoordMsg {
+    pub xid: u16,
+    pub mid: u16,
+    pub mtype: MsgType,
+}
+// Member Response
+//
```

Comment:
Use triple slash `///` for item doc comments

Reply:
Changed.